bot/code_review_bot/report/base.py

```diff
@@ -146,6 +146,9 @@ def pluralize(word, nb):
         else:
             comment = ""
 
+        if not all(revision.contains(issue) for issue in issues if issue.is_publishable):
```

Comment:
is_publishable is a method, not a property. Your test will always be true here as it tests the presend of the method...